amr-wind/wind_energy/ABLStats.cpp

```diff
@@ -108,6 +110,9 @@ void ABLStats::post_advance_work()
     }
 
     process_output();
+
+    delete m_pa_sfs;
+    delete m_pa_tsfs;
```

Comment:
```suggestion
```

Reply:
Tried addressing in https://github.com/Exawind/amr-wind/pull/290/commits/22cbdfe875f7c910a3fd1369c871a361c06ac0dd